source/content/guides/composer-convert.md

```diff
@@ -25,34 +36,69 @@ export site=yoursitename
 
 </Alert>
 
-## Checkout a New Branch
+## Add the Pantheon Integrated Composer Upstream in a New Local Branch
 
-You're about to make some massive changes to the codebase. We recommend you to do this work on a new branch, as it might take you some time to complete and rolling back can be complicated:
+You're about to make massive changes to the codebase. We recommend you to do this work on a new branch, as it might take you some time to complete and rolling back changes can be complicated:
 
 1. In your local terminal, change directories to your site project. For example, if you keep your projects in a folder called `projects` in the home directory:
 
   ```bash{promptUser:user}
   cd ~/projects/$site/
   ```
 
-2. Create the new branch:
+1. Add the Pantheon Drupal Upstream as a new remote, fetch the `ic` branch, and checkout to a new local branch based on it called `composerify`:
 
   ```bash{promptUser:user}
-  git checkout -b composify
+  git remote add ic git@github.com:pantheon-upstreams/drupal-project.git && git fetch ic && git checkout -b composerify ic/master
   ```
 
-  You can replace `composify` with a branch name of your choosing, but all following steps assume this name.
+  This upstream is a useful starting point for converting Drupal 8 sites to Composer, but it is a Drupal 9 upstream. To use it for Drupal 8, we will adjust the version numbers in a later step.
 
-## Set up a Multidev (Optional)
+  You can replace `composerify` with another branch name. If you do, remember to adjust the other examples in this doc to match.
+
+1. Copy your configuration from the default branch:
+
+  ```bash{promptUser:user}
+  git checkout master sites/default/config
+  git mv sites/default/config/* config
+  git rm -f sites/default/config/.htaccess
+  git commit -m "Pull in configuration from default branch"
+  ```
+
+1. Check for `pantheon.yml` settings you need to preserve by comparing your old codebase's `pantheon.yml` to the new `pantheon.upstream.yml`:
+
+  ```bash{promptUser:user}
+  git diff master:pantheon.yml pantheon.upstream.yml
+  ```
+
+   - If there are settings from `pantheon.yml` (shown with a `-` in the diff output), consider copying over your old pantheon.yml to preserve these settings:
+
+     ```bash{promptUser:user}
+     git checkout master pantheon.yml
+     git add pantheon.yml
+     git commit -m 'Copy my pantheon.yml'
+     ```
+
+   Note that a few settings must be left as they are in `pantheon.upstream.yml` for sites using Integrated Composer. These are:
 
-If your Pantheon account has access to [Multidev](/multidev), create a Mmultidev to push your new code to:
+     * `web_docroot`
+     * `build_step`
+
+### (Optional) Return to Drupal 8
```

Comment:
I've tried to move this to different parts of the doc, but none really feel right, and I'm not sure it belongs here either. Feel free to find a better place for this.